integration/querydsl/blaze-persistence-querydsl-expressions/src/main/java/com/blazebit/persistence/querydsl/AbstractBlazeJPAQuery.java

```diff
@@ -253,9 +239,11 @@ public String getQueryString() {
     }
 
     @Override
+    @SuppressWarnings({ "unchecked", "unsafe" })
     protected void clone(Q query) {
         super.clone(query);
         this.cachable = query.cachable;
+        this.binds.addBinds((List) query.binds.getArgs());
```

Comment:
Are `Operation` objects immutable? Otherwise you'd have to copy them as well

Reply:
Usually they should be, but there are some mutable expression types that theoretically could be referenced transitively. To be sure, I am pulling it through a clone visitor now.